code/modules/admin/topic.dm

```diff
@@ -5321,6 +5321,32 @@
 
 
 	// ----- Religion and stuff
+	else if(href_list["ashpaper"])
+		if(!check_rights(R_ADMIN))
+			return
+		var/obj/item/weapon/paper/P = locate(href_list["ashpaper"])
+		if(!istype(P))
+			message_admins("The target doesn't exist.")
+			return
+		if(!is_type_in_list(/obj/item/weapon/stamp/chaplain,P.stamped))
+			message_admins("That reference isn't for a paper with a chaplain stamp.")
+			return
+		var/ash_type = P.ashtype()
+		new ash_type(get_turf(P))
+		if(iscarbon(P.loc))
+			var/mob/living/carbon/C = P.loc
+			C.apply_damage(10,BURN,(pick(LIMB_LEFT_HAND, LIMB_RIGHT_HAND)))
```

Comment:
can't burn the hand holding it?

Reply:
This is how ashing paper works, it burns a random hand.

I suppose we could get active hand though.

Reply:
We don't have a function for this and in fact it's a huge pain to get the hand that's holding it because we've never had a reason for this before, normally we just use the active hand because that's what's touching something bad.

I think I'll just keep it the same as existing paper burn effect for simplicity.